chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,669 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Probably the simplest BIP in the world (the short story)\n",
+    "\n",
+    "Here we define and solve a very simple Bayesian inverse problem (BIP) using \n",
+    "CUQIpy. The purpose of this notebook is to introduce the basic concepts of\n",
+    "Bayesian inverse problems in a simple way and to use minimal CUQIpy code to\n",
+    "solve it.\n",
+    "\n",
+    "In the next notebooks (the long story), we discuss more details about setting up\n",
+    "this BIP in CUQIpy and provide many exercises to help the reader to explore \n",
+    "using CUQIpy in solving BIPs and think of slightly different BIP modeling \n",
+    "scenarios.\n",
+    "\n",
+    "\n",
+    "## <font color=#CD853F> Contents of this notebook: </font>\n",
+    "  * [0. Learning objectives](#r-learning-objectives)\n",
```

Comment:
Thanks! I fixed it now. This is a good point about what we need in terms of the notebook table of contents TOC. To accommodate both browsing from the book and working on VS code, it could be helpful to have the TOC repeated. But it is a bit of overkill. We can maybe put the main sections only at the top of the notebook and not subsections (which is consistent with some of the training material we have, eg, Gibbs notebook). We can talk about today about it

Reply:
Based on our discussion the other day, I removed subheaders from the `Contents of this notebook` section.

Reply:
I could not get the links to work either in https://amal-ghamdi.github.io/CUQI-Book-test/chapter01/Intro_example_short.html#r-solving-the-bip at the time of writing.

Reply:
Sorry the test book is now ahead of this branch, eventually all changes in the test book will be added on top of this branch. You can see this notebook in the test book https://amal-ghamdi.github.io/CUQI-Book-test/chapter01/Intro_example_short.html (no changes done to this notebook in particular in the test branch)